packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -97,7 +97,7 @@ export class MirrorNodeClient {
         [MirrorNodeClient.GET_NETWORK_FEES_ENDPOINT, [400, 404]],
         [MirrorNodeClient.GET_TOKENS_ENDPOINT, [400, 404]],
         [MirrorNodeClient.GET_TRANSACTIONS_ENDPOINT, [400, 404]],
-        [MirrorNodeClient.CONTRACT_CALL_ENDPOINT, [400,404,415,429,500]],
+        [MirrorNodeClient.CONTRACT_CALL_ENDPOINT, [404, 415, 500]],
```

Comment:
just note that this is default values and currently is being override by env variable:
`ETH_CALL_ACCEPTED_ERRORS`